apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -46,17 +47,24 @@ export class AITagView extends React.Component {
   }
 
   validateImage = async () => {
+    const MAX_FILE_SIZE = 5 * Math.pow(2, 20); // 5MB limit from AWS Rekognition
+    const MIN_DIMENSION_SIZE = 80; // 80px limit
     const imageId = get(this.props.entries.image.getValue(), 'sys.id');
-    if (!imageId) { return true; }
+    if (!imageId) { return ; }
 
     const file = await this.props.space.getAsset(imageId);
     const locale = this.props.locale;
     const contentType = get(file, `fields.file.${locale}.contentType`);
+    const details = get(file, `fields.file.${locale}.details`);
     // test if file extension is PNG/JPEG/JPG
-    const isImageValid = new RegExp(/image\/png|jpeg|jpg$/, 'i').test(contentType);
+    const isImageTypeValid = new RegExp(/image\/png|jpe?g$/, 'i').test(contentType);
```

Comment:
Please indicate the string should start with `image` with `^image`.

Capture group is needed to limit alternative (`|`):

```
> /image\/jpg|png/.test('png')
true
> /image\/(jpg|png)/.test('png')
false
```

Reply:
Good catch